README.md

```diff
@@ -14,6 +14,12 @@ Formal name is `Universally Unique Lexicographically Sortable Identifier`.
 $ gem install ruby-ulid
 ```
 
+## Uninstall
+
+```console
+$ gem uninstall ruby-ulid
+```
```

Comment:
Thanks for your PR!

But that is a that is a common knowledge of Ruby user, so needless I believe 🙇